### DISCUSSION TITLE
📝Comments of [argorithm/prefix-sum]

### DISCUSSION THREAD

**Discussion by giscus** [PresentKim/legacy-blog-sveltekit#17] [Announcements]
# 📝Comments of [argorithm/prefix-sum]

배열이 주어졌을 때 특정 구간의 합을 반복해서 구해야하는 경우에 사용할 수 있는 알고리즘

https://present.kim/posts/argorithm/prefix-sum

**Comment by PresentKim** [comment]
쓰다 말았었네..
